src/exercises/01-use-persist/hook.ts

```diff
@@ -1 +1,24 @@
-export const usePersist = () => null
+import * as React from 'react'
+
+export const usePersist = <TValue>(
+  key: string,
+  value: TValue,
+): [TValue, React.Dispatch<React.SetStateAction<TValue>>] => {
+  const [state, setState] = React.useState(() => {
+    try {
+      const storedValue = localStorage.getItem(key)
+      if (!storedValue) {
+        throw new Error()
```

Comment:
instead, we can return the initial value
```suggestion
        return value
```